### ISSUE TITLE
Incompatible with cmake 3.19

### ISSUE THREAD

**Issue by jonringer** [awslabs/aws-c-common#734]
```
-- Configuring done
CMake Error: install(EXPORT "aws-c-common-targets" ...) includes target "aws-c-common" more than once in the export set.
-- Generating done
```
Seems to build fine with cmake-3.18.4

related posts:
- stackoverflow: https://stackoverflow.com/questions/64507009/aws-cpp-sdk-fails-during-windows-build-due-to-installexport-foo-targets
- nixpkgs: https://github.com/NixOS/nixpkgs/pull/104637#pullrequestreview-537802100

**Comment by JackBoosY** [comment]
Related: microsoft/vcpkg#14716

**Comment by glaubitz** [comment]
This affects multiple AWS C/C++ projects such as ```aws-c-event-stream``` as well.

**Comment by JackBoosY** [comment]
This is a cmake issue, see discussion https://discourse.cmake.org/t/error-when-installing-targets-and-exports-in-3-19-1/2245/6

**Comment by jonringer** [comment]
This should be fixed in the 3.19.2 release of cmake, closing
